WaarpR66/src/main/java/org/waarp/openr66/context/task/ExecMoveTask.java

```diff
@@ -107,14 +107,13 @@ public void run() {
         prepareCommandExec.getPumpStreamHandler();
     ExecuteWatchdog watchdog = prepareCommandExec.getWatchdog();
     final LastLineReader lastLineReader = new LastLineReader(inputStream);
-    final Thread thread = new Thread(lastLineReader, "ExecRename" +
-                                                     session.getRunner()
-                                                            .getSpecialId());
-    thread.setDaemon(true);
-    Configuration.configuration.getExecutorService().execute(thread);
+    lastLineReader.setName("ExecRename" + session.getRunner().getSpecialId());
```

Comment:
Would a more descriptive name be better for debugging purpose ?
 I reckon `ExecRename` resemmbles too much a task name that does not exist (`ExecOutput`, `ExecMove` and `ExecRename`).
 Maybe `OutputReader` or just `LastLineReader` might be better suited (since it is what it does)...
 WDYT ?

Reply:
Agreed for LastLineReader